src/analysis/Environments.test.ts

```diff
@@ -621,7 +621,7 @@ describe('Environments', () => {
   describe('Disks - ', () => {
     it('Renders page correctly with a disk', async () => {
       const disk = generateTestDiskWithGoogleWorkspace();
-      asMockedFn(list).mockReturnValue(Promise.resolve([disk]));
+      asMockedFn(list).mockReturnValue(Promise.resolve(disk));
```

Comment:
Hmm.. this doesn't seem correct, list disks should still return a list?

Reply:
Oh yeah, my bad! I was trying a bunch of stuff to make the test pass and did not clean up 🙈